tests/test_ERA5.py

```diff
@@ -17,6 +17,7 @@
           2.6851, 0.3740, 1.0000, 0.4739, 0.0114, 0.0243, 0.0069, 310.7211, 916.1947)
 
 
+@pytest.mark.skip(reason="Hanging test")
```

Comment:
ok for now. The NcML work is meant to keep data on the server and should speed this up.

Reply:
Keep it for now, or remove and merge this PR later?